src/components/Nav/Nav.js

```diff
@@ -0,0 +1,367 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import TopBanner from './TopBanner';
+import { flexSet } from '../../styles/Variable';
+import styled from 'styled-components';
+
+function Nav() {
+  return (
+    <>
+      <TopBanner />
+      <MainNavContainer>
+        <GoToMainPageLink to="/">은행의집</GoToMainPageLink>
+        <LeftMenuCategoryWrapper>
+          <LeftMenuCategoriesSelected>커뮤니티</LeftMenuCategoriesSelected>
+          <LeftMenuCategories>스토어</LeftMenuCategories>
+          <LeftMenuCategories>인테리어시공</LeftMenuCategories>
+        </LeftMenuCategoryWrapper>
+        <RightMenuCategoryWrapper>
+          <SearchBoxWrapper>
+            <FaSearchIcon className="fas fa-search" />
+            <SearchBoxInput
+              aria-label="커뮤니티, 스토어 및 인테리어시공 검색"
+              placeholder="은행의집 통합검색"
+              type="search"
+            />
+            <SearchBoxButton>
+              <FaCameraIcon className="fas fa-camera" />
+            </SearchBoxButton>
+          </SearchBoxWrapper>
+          <CartIconWrapper>
+            <FaShoppingCartIcon className="fas fa-shopping-cart" />
+          </CartIconWrapper>
+          <LoginAndSignUpWrapper>
+            <GoToLoginPageLink to="/login">로그인</GoToLoginPageLink>
+            <GoToSignUpPageLink to="/signup">회원가입</GoToSignUpPageLink>
+          </LoginAndSignUpWrapper>
+          <WritingMenuWrapper>
+            <WritingMenuButton>
+              <GoToWritingPageLink to="/writing">글쓰기</GoToWritingPageLink>
+              <FaChevronDownIcon className="fas fa-chevron-down" />
+            </WritingMenuButton>
+          </WritingMenuWrapper>
+        </RightMenuCategoryWrapper>
+      </MainNavContainer>
+      <BottomNavContainer>
+        <CommunityMenuWrapper>
+          <GoToHomeLink to="/">홈</GoToHomeLink>
+          <GoToPhotoLink to="/photo">사진</GoToPhotoLink>
+          <GoToHousePartyLink to="/houseparty">집들이</GoToHousePartyLink>
+          {COMMUNITY_CATEGORY_LIST.map(communityCategory => {
+            return (
+              <CommunityMenuCategories key={communityCategory}>
+                {communityCategory}
+              </CommunityMenuCategories>
+            );
+          })}
+          <GoToQAndALink to="/questionandanswer">질문과답변</GoToQAndALink>
+        </CommunityMenuWrapper>
+        <RightSideMenuWrapper to="/">
+          <BankHouseLogo alt="BankHouse Logo" src="/images/logoImage.png" />
+          <BetaVersionInterior>3D인테리어(BETA)</BetaVersionInterior>
+        </RightSideMenuWrapper>
+      </BottomNavContainer>
+    </>
+  );
+}
+
+const COMMUNITY_CATEGORY_LIST = [
+  '노하우',
+  '전문가집들이',
+  '셀프가이드',
+  '이벤트',
+];
+
+const MainNavContainer = styled.div`
+  ${flexSet('start', 'center')};
```

Comment:
mixin함수 만드실때 기본값 center로 적용되어 있는 것 같았는데 그러면 
```suggestion
  ${flexSet('start')};
```
두번째 인자부분은 default가 center니까 따로 인자 전달 안해줘도 center가 들어가겠죠?